buildroot-external/patches/occu/0153-WebUI-DynamicResizing.patch

```diff
@@ -0,0 +1,126 @@
+--- occu/WebUI/www/rega/login.htm.orig
++++ occu/WebUI/www/rega/login.htm
+@@ -48,31 +48,10 @@
+           FormSubmit();
+         }
+       }
+-
+-      recalcDivs = function()
+-      {
+-        var wHeight = 0;
+-        if (window.innerHeight)
+-        {
+-          wHeight = window.innerHeight;
+-        }
+-        else if (document.documentElement.offsetHeight)
+-        {
+-          wHeight = document.documentElement.offsetHeight;
+-        }
+-
+-        var wHeaderHeight = 72;
+-        var wFooterHeight = 32;
+-        var wContentHeight = wHeight - wHeaderHeight - wFooterHeight - 27;
+-
+-        $("navbar").setStyle({height: wHeaderHeight + "px"});
+-        $("content").setStyle({height: wContentHeight + "px"});
+-        $("footer").setStyle({height: wFooterHeight + "px"});
+-      }
+     </script>
+   </head>
+   <body>
+-    <table style="width:100%; height:100%;" cellpadding="0" cellspacing="0" id="maintable">
++    <table style="width:100vw; height:100vh;" cellpadding="0" cellspacing="0" id="maintable">
```

Comment:
If we on changing the style anyway. Why not remote these element-specific `style=` definitions here alltogether and better put them into the global `style.css`?

Reply:
Well, one could introduce a new tag `#maintable` in `style.css` and do all the styling from there. Let me think ...

Reply:
fixed